CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-# How to Contribute
+# How to Contribute in 2.0-preview
```

Comment:
@TrevorJoelHarris  I know you were looking into making changes to Contributing guidelines. Tagging in case you have feedback based on that parallel effort.

Reply:
Yes, I did add some internal only guidelines for contributing to our wiki. I think for now there's nothing to do here. Ultimately I'd like to take a swing at standardizing the internal and external guidance but I think that will involve some careful work that requires us making sure we've crossed all our t's and dotted all our i's.

This seems very targetted to "how to check in to this repo" which seems okay. I don't think there's any need to converge the two before angela checks in